test/integration/next-image-legacy/trailing-slash/test/index.test.ts

```diff
@@ -54,7 +54,6 @@ describe('Image Component Trailing Slash Tests', () => {
     afterAll(() => killApp(app))
 
     it('should include trailing slash when trailingSlash is set on config file during next start', async () => {
-      expect.assertions(1)
```

Comment:
Any reason this was removed?

Reply:
Yeah it's using `check()` so `expect()` can be called more than once which causes this flake when `check()` actually retries.

Reply:
Ideally we should move the expect outside of the check() and outside of the try/catch so that its called exactly once.